source/server/filter_chain_manager_impl.cc

```diff
@@ -600,6 +600,7 @@ const Network::FilterChain* FilterChainManagerImpl::findFilterChainForSourceIpAn
 
 void FilterChainManagerImpl::convertIPsToTries() {
   for (auto& [destination_port, destination_ips_pair] : destination_ports_map_) {
+    UNREFERENCED_PARAMETER(destination_port);
```

Comment:
scary folding.. but I think this one can be _ and leave the rest marked by UNREFERENCED_PARAMETER to avoid var name redelcare